<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TuontiMaaritys</class>
 <widget class="QWidget" name="TuontiMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tuonti</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ostolaskuGroup">
     <property name="title">
      <string>Pdf-ostolaskujen tuonti</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Kirjausperuste</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="perusteCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vastatili</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TilinvalintaLine" name="velkatiliEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="tositelajiCombo"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tositelaji</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tilioteRadio">
     <property name="text">
      <string>Pdf-tiliotteiden tuonti</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>128</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header>db/tilinvalintaline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
